stories/components/Modals.js

```diff
@@ -475,24 +477,62 @@ storiesOf('Components', module).add('Modals', () => (
       </ModalTrigger>
     </StoryItem>
 
-    <StoryItem
-      title="Modal with ImageHeader"
-      description="Modal with ImageHeader to display an image."
-    >
-      <ModalTrigger>
-        <Modal.Container>
-          <Modal.ImageHeader height={401} imageUrl="https://metro.co.uk/wp-content/uploads/2017/07/187144066.jpg?quality=90&strip=all" />
-          <Modal.Body>Look at this lovely image ☝️</Modal.Body>
-          <Modal.Footer spaceBetween>
-            <Button types={['link']} clickHandler={action('clickHandler')}>
-              Cancel
-            </Button>
-            <Button types={['primary']} clickHandler={action('clickHandler')}>
-              Confirm
-            </Button>
-          </Modal.Footer>
-        </Modal.Container>
-      </ModalTrigger>
-    </StoryItem>
-  </div>
+        <StoryItem
+            title="Modal with ImageHeader"
+            description="Modal with ImageHeader to display an image."
+        >
+            <ModalTrigger>
+                <Modal.Container>
+                    <Modal.ImageHeader height={401}
+                                       imageUrl="https://metro.co.uk/wp-content/uploads/2017/07/187144066.jpg?quality=90&strip=all"/>
+                    <Modal.Body>Look at this lovely image ☝️</Modal.Body>
+                    <Modal.Footer spaceBetween>
+                        <Button types={['link']} clickHandler={action('clickHandler')}>
+                            Cancel
+                        </Button>
+                        <Button types={['primary']} clickHandler={action('clickHandler')}>
+                            Confirm
+                        </Button>
+                    </Modal.Footer>
+                </Modal.Container>
+            </ModalTrigger>
+        </StoryItem>
+
+        <StoryItem
+            title="Modal with ModalHeaderNavigation"
+            description="Modal with tabs in the header."
+        >
+            <ModalTrigger>
+                <Modal.Container size="large">
+                    <Modal.HeaderNavigation title="Title!">
+                        <Navigation tabs>
+                            <MenuItem active onClick={action('clickHandler')}>
+                                {/* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */}
+                                <div className="h-full w-full" role="menuitem" tabIndex={0} >
```

Comment:
What do you think to making these buttons rather than divs? Or is a MenuItem already a button? Is this purely for the tabindex?

Reply:
This is purely for a tabindex, `MenuItem` is an anchor that doesn't have a href, and we only need the `onClick` for it.